src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryClientExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.TelemetryCore
+{
+    /// <summary>
+    /// Extension methods used for telemetry client.
+    /// </summary>
+    internal static class TelemetryClientExtensions
```

Comment:
For the two methods in this class I would change them to extend an `ITelemetryClient[]`.